docs/tutorials/embedding/AdjacencySpectralEmbed.ipynb

```diff
@@ -19,9 +19,9 @@
     "np.random.seed(8889)\n",
     "from sklearn.metrics import adjusted_rand_score\n",
     "from sklearn.cluster import KMeans\n",
-    "from graspy.embed import AdjacencySpectralEmbed\n",
-    "from graspy.simulations import sbm\n",
-    "from graspy.plot import heatmap, pairplot\n",
+    "from graspologic.embed import AdjacencySpectralEmbed\n",
```

Comment:
This explodes when I ran sphinx-build locally.  Not sure what is wrong about it, but...

Reply:
works fine in the netlify build so not sure how to help

Reply:
Yeah, that threw me for a loop yesterday.  Fresh venv and everything.  I don't know how the nbsphinx stuff works so I don't know that I'll dig into it for now - since Netlify is fine with it, I'm not too concerned about it (and it also didn't fail in github actions, so /shrug)